### PR TITLE
#160 book ratings are now displayed

### LINKED ISSUE #160: Make the front end retrieve ratings data from the back-end
Make the front-end load and display rating information from the back end.
Information will need to be loaded in two places:

- [x] When displaying a review in focus mode the ratings from that review should be displayed
- [x] The book details should display the average of the ratings for each genre

### PR DESCRIPTION
**Pull request by CameronJung**
- average ratings now display
- ratings from reviews now display in focus mode
closes #160

### CODE REVIEW

File: bookwyrm_frontend/src/views/BookDetail.vue [hunk 1]
```diff
@@ -70,7 +70,7 @@
           <!-- Avg Rating -->
           <RatingComponent
             v-bind:displayOnly="true"
-            v-bind:initialRatings="ratingsData"
+            v-bind:ratings="bookDetails.avg"
```
Comment (LukeBMorrow) [inline bookwyrm_frontend/src/views/BookDetail.vue:73]: I believe it should be bookDetails.avgRate, though if it works for your manual tests and your branch is caught up to main, then disregard this.

**Review by LukeBMorrow — COMMENTED** [review]
I think there might be an issue, though you'd have to let me know if you've tested it manually

File: bookwyrm_frontend/src/views/BookDetail.vue (hunk 1 above)
Comment (CameronJung) [reply bookwyrm_frontend/src/views/BookDetail.vue:73]: It shouldn't be it uses the accessor not the variable and the accessor is called getavg()

Comment (CameronJung) [reply bookwyrm_frontend/src/views/BookDetail.vue:73]: I have tested it manually, I actually always do manual tests.

Comment (LukeBMorrow) [reply bookwyrm_frontend/src/views/BookDetail.vue:73]: We'll have to fix that at some point. Just be sure to check for console errors as well.

**Review by LukeBMorrow — APPROVED** [review]
Looks good

**Pull request merged**